src/javaRosa.js

```diff
@@ -1606,12 +1609,14 @@ define([
                         for (var k = 0; k < forms.length; k++) {
                             form = forms[k];
                             val = form.getValueOrDefault(lang);
-                            xmlWriter.writeStartElement("value");
-                            if(form.name !== "default") {
-                                xmlWriter.writeAttributeString('form', form.name);
+                            if (val) {
```

Comment:
Is there ever a case where `val` evaluates to false, and therefore the `<value></value>` tag is not written? That was the bug we hit [last time](https://github.com/dimagi/Vellum/commit/ccd3475d298337f954034c4e1d8d37ce8e04dfbd), right?